packages/react-native-aztec/android/gradlew

```diff
@@ -205,6 +205,12 @@ set -- \
         org.gradle.wrapper.GradleWrapperMain \
         "$@"
 
+# Stop when "xargs" is not available.
+if ! command -v xargs >/dev/null 2>&1
+then
+    die "xargs is not available"
+fi
+
```

Comment:
This file was automatically created when updating the Gradle wrapper (https://github.com/WordPress/gutenberg/pull/51289/commits/d108e4d70456c87ba0cdb953951e1596ad81072f). The wrapper is present in the three RN packages `react-native-aztec`, `react-native-bridge`, and `react-native-editor`. I think we don't see it in the upgrade helper because these files are ignored ([reference](https://github.com/facebook/react-native/blob/d097408c1ca15d3aa9dd69dc12f1043f54890213/.gitignore#L33-L39)).